src/js/components/Box/index.d.ts

```diff
@@ -23,7 +24,7 @@ export interface BoxProps {
   elevation?: "none" | "xsmall" | "small" | "medium" | "large" | "xlarge" | string;
   flex?: "grow" | "shrink" | boolean | {grow?: number,shrink?: number};
   fill?: "horizontal" | "vertical" | boolean;
-  gap?: "xxsmall" | "xsmall" | "small" | "medium" | "large" | "xlarge" | string;
+  gap?: GapType | "xxsmall" | "xsmall" | "xlarge";
```

Comment:
we seem to be repeating this a lot. only distribution seems different so how about we support in every component the same gaps in order to avoid this?

Reply:
I agree, it seems like `"xsmall" | "xlarge"` is also common to all gaps, any reason we are not including it as well?

Reply:
@ShimiSun I think the reason they were not included in all components was because of design decisions that "xxsmall" gap may not look correct on certain components? That's what I interpreted from our discussion yesterday. 

However, "xxsmall" - "xlarge" is defined for all components, so if it is okay I can adjust GapType to include the full range and then update the docs for each of the components. Let me know if this is the direction we want to move in.

Reply:
But only component missing is distribution right? maybe we can add it? i didn't mean all the components of grommet, just the ones implementing already gap :)

Reply:
I understand now. Yes, I can add it into distribution.